dask-gateway/Dockerfile

```diff
@@ -1,95 +1,36 @@
-# ** A base miniconda image **
-FROM debian:bullseye-slim as miniconda
-LABEL MAINTAINER="Jim Crist-Harif"
-
-# List of conda versions: https://repo.anaconda.com/miniconda/
+# USAGE DISCLAIMER:
```

Comment:
Is this text a disclaimer?

Reply:
Hmmmm I'm not sure. Googling the definition of disclaimer led me to:

> a statement that denies something, especially responsibility.

This is the part that I was thinking of as a disclaimer, it is quite indirect, it is a suggestion that we don't want to be held responsible for meeting the needs of the users by asking them to build their own.

> Please build your own according to
# the documentation if this very limited image doesn't meet your needs.

---

I'm not confident on a change to make here but I'm very happy to accept any change suggestions to this text.

Reply:
```suggestion
# USAGE:
```

(or perhaps "purpose", "description". To me "disclaimer" sounds like "warning".